packages/salesforcedx-vscode-core/src/commands/forceApexTestRun.ts

```diff
@@ -81,35 +82,59 @@ export class TestsSelector
   }
 }
 
+export class ForceApexTestRunCommandFactory {
+  private _data: ApexTestQuickPickItem;
```

Comment:
Let's drop the _ from the variable names. This is not a convention that we use for private fields.